app/src/main/java/org/metabrainz/android/presentation/features/critiques/CritiqueActivity.kt

```diff
@@ -0,0 +1,193 @@
+fun ReviewCard(review: Review, name:String, modifier: Modifier) {
+    Card(
+        modifier = modifier
+            .padding(vertical = 4.dp, horizontal = 8.dp)
+            .clip(RoundedCornerShape(16.dp)),
+        backgroundColor = MaterialTheme.colors.onSurface
+
+    ) {
+        Row(modifier = modifier.padding(24.dp)) {
+            Column(
+                modifier = modifier
+                    .weight(1f)
+            ) {
+                Text(
+                    text = review.id,
+                    modifier = Modifier.padding(0.dp, 0.dp, 12.dp, 0.dp),
+                    color = MaterialTheme.colors.surface,
+                    fontWeight = FontWeight.Bold,
+                    style = MaterialTheme.typography.subtitle1,
+                    textAlign = TextAlign.Center,
+                )
+                Spacer(modifier = Modifier.height(8.dp))
+                Text(
+                    text = "Review by: $name",
+                    modifier = Modifier.padding(0.dp, 0.dp, 12.dp, 0.dp),
+                    color = MaterialTheme.colors.surface,
+                    style = MaterialTheme.typography.caption,
+                    textAlign = TextAlign.Center
+                )
+            }
+        }
+    }
+}
+
+@Composable
+fun EntityChips(
+    viewModel: CritiquesViewModel = viewModel(),
+    activity: Activity
+) {
+    val entities = listOf(AllReviews,Artist,ReleaseGroup, Label, Recording, Event, Place, Work)
+    viewModel.fetchAllReviews()
+    var selectedChipIndex by remember {
+        mutableStateOf(0)
+    }
+    LazyRow(
+        verticalAlignment = Alignment.Top,
+        horizontalArrangement = Arrangement.Center
+    ) {
+        items(entities.size) {
+            Box(modifier = Modifier
+                .padding(start = 15.dp, top = 15.dp, bottom = 15.dp)
+                .clickable {
+                    if (App.context?.isOnline == false) {
+                        Toast
+                            .makeText(
+                                activity,
+                                "Connect to Internet and Try Again",
+                                Toast.LENGTH_LONG
+                            )
+                            .show()
+                        return@clickable
+                    }
+                    selectedChipIndex = it
+                    viewModel.reviews = listOf()
+                    viewModel.fetchReviews(entities[it].entity)
+                }
+                .clip(RoundedCornerShape(30.dp))
+                .background(
+                    if (selectedChipIndex == it)
+                        colorResource(
+                            id = R.color.mb_purple_medium
+                        )
+                    else colorResource(id = R.color.light_gray)
+                )
+                .padding(12.dp)) {
+                Row {
+                    Text(
+                        text = entities[it].displayName,
+                        color = Color.Black,
```

Comment:
I think white color would be a better preference here.

Reply:
color changed to white